### PR TITLE
make sure python & pip are installed

### LINKED ISSUE #98: Execution environments no longer build
See https://github.com/ansible-community/images/actions/runs/11660326522/job/32462440567#step:5:65:
```
[1/4] STEP 13/14: RUN /output/scripts/pip_install $PYCMD
+ PYCMD=/usr/bin/python3
+ '[' -z /usr/bin/python3 ']'
+ '[' '!' -x /usr/bin/python3 ']'
/usr/bin/python3 is not an executable
+ echo '/usr/bin/python3 is not an executable'
+ exit 1
Error: error building at STEP "RUN /output/scripts/pip_install $PYCMD": error while running runtime: exit status 1
```
I guess it's now necessary to explicitly install Python 3 in the image.

### PR DESCRIPTION
**Pull request by eRadical**
fixes #98

### CODE REVIEW

**Comment by samccann** [comment]
Thanks for this @eRadical !  Looks good to me but I don't have the technical chops to officially approve.  I added a few reviewers that hopefully can get this reviewed and merged.

**Review by Andersson007 — APPROVED** [review]

**Comment by Andersson007** [comment]
@felixfontein if you're OK, please merge

**Comment by eRadical** [comment]
Thank you @samccann! No rush...

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 execution-environments/*/context
 execution-environments/*/ansible-navigator.log
 execution-environments/*/*artifact*.json
+.idea/*
```
Comment (felixfontein) [inline .gitignore:4]:
```suggestion
/.idea/
```

**Comment by felixfontein** [comment]
@eRadical thanks for your contribution!
@Andersson007 thanks for reviewing!

**Pull request merged**